src/common/darktable.c

```diff
@@ -78,7 +78,9 @@
 #include <sys/wait.h>
 #endif
 #include <locale.h>
+#if defined(__SSE__)
```

Comment:
These ifdefs around xmmintrin.h could probably be factored out into a common header file.